src/mcl_3dl.cpp

```diff
@@ -299,12 +299,22 @@ class MCL3dlNode
 
     try
     {
-      sensor_msgs::PointCloud2 pc2_tmp;
-      pcl::toROSMsg(*pc_local_accum_, pc2_tmp);
-      geometry_msgs::TransformStamped trans = tfbuf_.lookupTransform(
-          frame_ids_["base_link"], pc2_tmp.header.frame_id, pc2_tmp.header.stamp, ros::Duration(0.1));
-      tf2::doTransform(pc2_tmp, pc2_tmp, trans);
-      pcl::fromROSMsg(pc2_tmp, *pc_local_accum_);
+      const geometry_msgs::TransformStamped trans = tfbuf_.lookupTransform(
+          frame_ids_["base_link"],
+          pc_local_accum_->header.frame_id,
+          ros::Time().fromNSec(pc_local_accum_->header.stamp * 1000), ros::Duration(0.1));
```

Comment:
[PCLHeader::timestamp](http://docs.pointclouds.org/1.8.1/structpcl_1_1_p_c_l_header.html#a7ccf28ecce53332cd572d1ba982a4579) is in microseconds

Reply:
```cpp
ros::Time pcl_conversions::fromPCL(const pcl::uint64_t& pcl_stamp)
```
http://docs.ros.org/jade/api/pcl_conversions/html/namespacepcl__conversions.html#a18dc9e2742a669634527aebbb8853336